### DISCUSSION TITLE
mempool manipulation?

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#1023] [General]
does the mempool for chia work pretty much the same as bitcoin's?

a scenario crossed my mind recently... if the mempool gets to the point where it is not consistently emptying, could miners force up transaction fees by generating a lot fake of transactions at low fees. then maybe even cancel the transactions if the mempool starts shrinking?

what kind of game theory has been considered in the mempool world? thanks!

**Comment by Stumble** [comment]
wouldn't it the same as other blockchain projects? mempool picks up transactions sorted by fees, as long as signature is correct.

**Reply by hoffmang9** [reply]
This is correct.

You can do some fancier things with chialisp and smart coins to control how a given transaction can go through though.

**Comment by dorkmo** [comment]
not mempool per se but i thought this was in a similar vein.

https://reut.rs/3lEDM67
